tests/integration/questionnaire/test_questionnaire_placeholders.py

```diff
@@ -34,6 +34,11 @@ def test_title_placeholders_rendered_in_summary(self):
 
     def test_placeholders_rendered_in_pages(self):
         self.launchSurvey("test_placeholder_transform")
+        self.assertInBody(
+            'If the company details or structure have changed contact us on <a href="tel:03001234931">0300 1234 931</a> '
+            'or email <a href="mailto:surveys@ons.gov.uk?subject=Change%20of%20details%20reference%20123456789012A">surveys@ons.gov.uk</a>'
+        )
```

Comment:
It seems unnecessary to duplicate the schema from `test_introduction.json` into `test_placeholder_transform.json` just so it can be tested here. Can a test be added that uses `test_introduction.json` instead?

Reply:
We perhaps need to have a rethink on test_placeholder_transform.json as I instinctively thought all our transforms would live in there, perhaps they should

Reply:
Yeah, it's not clear, as there is also the `test_placeholder_full` schema that also tests placeholder transforms. I think as it already exists in the `test_introduction` it isn't needed in either of these schemas.